templates/model_simple.mustache

```diff
@@ -13,17 +13,19 @@ type {{classname}} struct {
 {{^-first}}
 {{/-first}}
 {{#description}}
-	// {{{.}}}
+	// {{#deprecated}} Deprecated: {{/deprecated}}{{{.}}}
```

Comment:
I think that we should't use the description as a deprecation message, it may look odd. Have you tested this? It's not one of the cases you showed in screenshots.

I think we could just display the deprecated message `// Deprecated: Check the GitHub changelog for alternatives` and then the description, just like the endpoints

Reply:
Currently there is no field in any OAS that is deprecated _and_ has a description. But I would assume the teams will use it as they use the endpoints: when marking a field as deprecated, update the description with the reason

Reply:
In the endpoints I am also using the description, and all teams with deprecated endpoints have a deprecation message in the first line of the description, followed by `"\n {rest of description}"`

Reply:
That makes sense, if there are no current use-cases we can investigate the cases as they come